src/templates/base.html

```diff
@@ -240,4 +275,14 @@ <h4 class="ui inverted header">Footer Header</h4>
     {% endblock %}
 
 </body>
+<style>
```

Comment:
can we do this in a stylus file somewhere? so we don't have to have this css in base.html?